app/email_batching.go

```diff
@@ -68,12 +71,17 @@ func NewEmailBatchingJob(a *App, bufferSize int) *EmailBatchingJob {
 }
 
 func (job *EmailBatchingJob) Start() {
-	if task := model.GetTaskByName(EMAIL_BATCHING_TASK_NAME); task != nil {
```

Comment:
This is the only usage of `GetTaskByName`, so storing the task in the job (which is only created once per app) instead allows us to get rid of some global state.